### ISSUE TITLE
Add error to `--server-data-dir` if path is too long in `run_rstudio_singularity.sh`

### ISSUE THREAD

**Issue by lianov** [U-BDS/training_guides#17]
If users are running RStudio containers within heavily nested directories with `--server-data-dir` enabled, a rserver error may show indicating path names are too long. For example passing if `--server-data-dir` from `run_rstudio_singularity.sh` within a directory that is of length 75, the following occurs:

```
2022-06-09T23:25:59.800591Z [rserver] ERROR Unexpected exception: File name too long; LOGGED FROM: void rstudio::core::http::AsyncServerImpl<ProtocolType>::handleConnection(rstudio_boost::shared_ptr<rstudio::core::http::AsyncConnectionImpl<typename ProtocolType::socket> >, rstudio::core::http::Request*) [with ProtocolType = rstudio_boost::asio::ip::tcp  typename ProtocolType::socket = rstudio_boost::asio::basic_stream_socket<rstudio_boost::asio::ip::tcp>] src/cpp/server/ServerInit.cpp:575
```

It's worth noting that this seems limited to `--server-data-dir` instead of standard binding via `-b`, and thus a limitation only within `rserver`

To prevent this error and aid users in a more straight-forward manner, we should add a check of length for the current working directory and add an error message indicating that users will have to move to upstream directories or run the job from `$USER_SCRATCH` (ideally we should also find the upper limit in this length).

**Comment by atrull314** [comment]
Went ahead and added a quick update to the script, still need to find what the upper limit is, but I have set it to 75 for the moment.

**Comment by lianov** [comment]
As an fyi - the commit has been merged to main (PR #19) ; but I will for [now] leave this issue open until we either 1. find the upper limit for `rserver` or 2. re-assess that 75 is fully sufficient to solve the any issues that users have encountered (pending user-feedback).

**Comment by lianov** [comment]
This has been left open for a while to see if anyone else encountered an issue since the last update. So far it has been fine, thus will finally close it. If needed we can open another issue in the future.
